### ISSUE TITLE
Add localized automation name properties in WinUI

### ISSUE THREAD

**Issue by mvegaca** [microsoft/WindowsTemplateStudio#4066]
**Describe the bug**
Add localized automation name properties in WinUI.
The way to set this automation names from Resw dictionary from xaml does not work in WinUI 3 Preview 4.

**xaml definition**
```xml
<Button x:Uid="WebView_RefreshBrowserButton" />
```

**resw definition**
```xml
<data name="WebView_RefreshBrowserButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
  <value>Refresh</value>
</data>
```

By this moment I suggest to add the localized resource in the Resw as a normal string value and consume it from the page code behind file.

```csharp
AutomationProperties.SetName(ReloadButton, "WebView_ReloadButtonAutomationPropertiesName".GetLocalized());
```

Applies to the following platforms:
| UWP              | WPF              | WinUI           |
| :--------------- | :--------------- | :---------------|
| No | No | Yes |

**Comment by mvegaca** [comment]
I've verified that setting AutomationProperties Name is not working in WinUI 3 Preview 4. I'm going to prepare a PR with the code behind assignment workaround.

**Comment by sibille** [comment]
Verified in dev-nightly:
Templates version: 0.22.21064.1
Wizard version: 0.22.21064.1
